projects/aca-content/src/lib/aca-content.routes.ts

```diff
@@ -562,7 +818,12 @@ export const CONTENT_LAYOUT_ROUTES: Route = {
       children: [
         {
           path: '',
-          loadChildren: () => import('@alfresco/aca-folder-rules').then((m) => m.AcaFolderRulesModule)
```

Comment:
you are killing application performance and bundling for no particular reason. this must be lazy loaded on demand

Reply:
reverted